index.html

```diff
@@ -7,10 +7,92 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+<!-- Main Content -->
+<main>
+<header>
+<div class="header">
+    <div class="container">
+        <img class="logo" src="/images/logo.png" alt="logo">
+    </div>
+    <div class="container-right">
+        <h1>Your Cake Mate</h1>
+        <h2>For Your Orders <br> 07403720162</h2>
+    </div>
+</header>
+<nav>  
+  <div class="topnav">
+    <a href="#">Home</a>
+    <a href="#">Cakes</a>
+    <a href="#">Ordering</a>
+    <a href="#">Lessons</a>
+    <a href="#">About</a>
+  </div>
+</nav> 
+
+
+    <div class="row">
+        <div class="rightcolumn">
+            <div class="card">
+                <h2>CAROUSSEL</h2>
+                <div class="img" style="height:200px;">Image</div>
+                <p>Prefer to do it the old fashioned way?<br>
+                    We have stores all across London so why not pop in, say, 'Hi' and pick up a box of your favourites!
+                    Click here to find my nearest store/collection point <br></p>
+            </div>
+        </div>
+        <div class="leftcolumn">
+            <div class="card">
+                <h2>Hello</h2>
+                <img class="img" src="/images/selfridges2.jpg" alt="shop"></div>
```

Comment:
Try to remember to keep the indentation consistent for elements that are nested -  so the div at the end of this line would be on the next line so that its clear where a html element begins and ends. Have another look at this section - there is an extra `</div>` element here.